Resources/Prototypes/Entities/Objects/Tools/appraisal.yml

```diff
@@ -20,4 +20,6 @@
       - Belt
     - type: Tag
       tags:
-      - AppraisalTool
\ No newline at end of file
+      - AppraisalTool
+    - type: StaticPrice
+      price: 65
```

Comment:
This should not have a price as it's lathe manufacturable and spammable.

Reply:
done